docs/en/pid-eaa-issuance.rst

```diff
@@ -673,10 +735,63 @@ If the authentication is successful the PID/(Q)EAA Issuer redirects the User by
       - Unique identifier of the PID/(Q)EAA Issuer who created the Authentication Response. The Wallet Instance MUST validate this parameter.
       - [:rfc:`9207`], [:rfc:`7519`, Section 4.1.1.].
 
-If any errors occur during the Authorization Request, the Authorization Server MUST return an error response as defined in :rfc:`6749#section-4.1.2.1`. The response MUST use *application/json* as the content type and MUST include the following parameters:
+If any errors occur during the Authorization Request, the Authorization Server MUST return an error response as defined in :rfc:`6749#section-4.1.2.1`. 
+In case of invalid/missing ``redirect_uri`` or ``client_id`` Authorization Server MUST NOT redirect the User to the redirection URI and MUST inform him of the error.
```

Comment:
```suggestion
In case of invalid/missing ``redirect_uri`` or ``client_id`` Authorization Server MUST inform the User with the error and MUST NOT redirect the User to the redirection URI.
```